src/applications/personalization/profile/tests/components/contact-information/ContactInformation.unit.spec.jsx

```diff
@@ -54,9 +54,13 @@ const testContactInfo = () => {
 
   // It's too cumbersome to convert the raw phone number data into what is
   // displayed so I'm using strings here.
-  expect(view.getByText('555-555-5559', { exact: false })).to.exist;
-  expect(view.getByText('804-205-5544, ext. 17747')).to.exist;
-  expect(view.getByText('214-718-2112', { exact: false })).to.exist;
+
+  // Checking that va-telephone shadow dom values exist is done with Cypress at:
+  // personalization/profile/tests/e2e/contact-information/
+
+  // expect(view.getByText('555-555-5559', { exact: false })).to.exist;
+  // expect(view.getByText('804-205-5544, ext. 17747')).to.exist;
+  // expect(view.getByText('214-718-2112', { exact: false })).to.exist;
```

Comment:
RTL does not work for testing shadow dom values so these tests are being commented out. The telephone values are already being tested in a related Cypress test though so these unit tests seem redundent. `personalization/profile/tests/e2e/contact-information/`